app/src/main/res/layout/screen_patients.xml

```diff
@@ -10,7 +10,7 @@
   <org.simple.clinic.widgets.OmegaCenterIconButton
     android:id="@+id/patients_search_patients"
     android:layout_width="match_parent"
-    android:layout_height="56dp"
```

Comment:
You can leave this file unedited for the time being.
Once your feature completes these changes will anyway be merged from your `screen_patients_v2`